lib/base_service.js

```diff
@@ -107,35 +107,50 @@ BaseService.prototype._getOptions = function(opts) {
             global: true
         }
     })
-    .command('docker-start', 'starts the service in a Docker container')
-    .command('docker-test', 'starts the test process in a Docker container')
-    .command('build', 'builds the service\'s package and deploy repo', function(yargs) {
-        return yargs.option({
-            f: {
-                alias: 'force',
-                default: false,
-                describe: 'force the operation to execute',
-                type: 'boolean'
-            },
-            d: {
-                alias: 'deploy-repo',
-                default: false,
-                describe: 'build only the deploy repo',
-                type: 'boolean'
-            },
-            s: {
-                alias: 'reshrinkwrap',
-                default: false,
-                describe: 'rebuild shrinkwrap.json by removing and regenerating after npm install',
-                type: 'boolean'
-            },
-            r: {
-                alias: 'review',
-                default: false,
-                describe: 'send the patch to Gerrit after building the repo',
-                type: 'boolean'
-            }
-        });
+    .command('docker-start', 'starts the service in a Docker container', {
+        nobuild: {
+            default: false,
+            describe: 'do not build nor execute the container, only generate the Dockerfile spec',
+            type: 'boolean'
+        }
+    })
+    .command('docker-test', 'starts the test process in a Docker container', {
+        nobuild: {
```

Comment:
IMHO, `docker-build` just complicates things because it creates confusion with the existing `build` command, and building the container after having the Dockerfile is a one-liner, so it can be easily scripted away.